### DISCUSSION TITLE
为什么chooseAvatar在真机无法触发弹窗呢

### DISCUSSION THREAD

**Discussion by kuonji-arisu** [NervJS/taro#13907] [Q & A]
如题，我的taro，开发者工具和基础库都是最新的（2.23.0, IPhone13 mini）
![image](https://github.com/NervJS/taro/assets/29818319/db21be98-227b-4be7-af01-e44e34a82e6c)
一开始开发和真机测试都是正常的，
然后某一天开始，在真机调试的时候，选择头像的弹窗就打不开了
本来想搞一个最小环境提交issue，结果那个最小环境在同样的设置下居然是正常的...

**Comment by TheKonka** [comment]
真机调试？试下预览呢？ 弹窗打不开vconsole有报错吗？

**Reply by kuonji-arisu** [reply]
真机/预览试了下都不行，而且vconsole一点错误都没有，canIUse返回的也是true，页面模板和逻辑全删到就剩一个button也还是不行。
我本来怀疑是最新基础库的问题，但是拆出来的最小运行环境却可以在真机上运行
